dashboard/app/models/script.rb

```diff
@@ -678,7 +678,15 @@ def has_standards_associations?
   end
 
   def standards
-    stages.map(&:standards).flatten.uniq.map(&:summarize)
+    standards = stages.map(&:standards).flatten.uniq
+    standards_with_stages = []
+    standards.each do |standard|
+      standards_summary = standard.summarize
```

Comment:
nit - I would call it `standard_summary` since this variable only holds the summary of one standard at a time